docs/faq/general/what-languages-can-be-used-with-metabase.md

```diff
@@ -17,9 +18,9 @@ The available languages as of the 0.35 release are:
 - Persian
 - Polish
 - Portuguese
-- Spanish
 - Russian
 - Slovak
+- Spanish
```

Comment:
Turkish was added back in a 0.35.x release.

Reply:
It was added in 0.35.4, but I'll add it - I would prefer not having a list like this, which contains version information, since it would have to be updated for each version (including minor). Could we do it some other way?